ckanext/dcat/utils.py

```diff
@@ -337,3 +339,16 @@ def parse_accept_header(accept_header=''):
                 return accepted_media_types_wildcard[_type]
 
     return None
+
+
+def parse_date_iso_format(date):
+    '''
+    Parses the supplied date and tries to return it as a string in iso format
+    '''
+    try:
+        date = date.replace('Z', '')
+        default_datetime = datetime.datetime(2000, 1, 1, 0, 0, 0)
+        _date = parse_date(date, default=default_datetime)
+        return _date.isoformat()
+    except:
```

Comment:
we definitely need some log message here.
and it is better to catch the only Exception instead of empty 'except'.

Reply:
I'll update catching exceptions, but the log messages are not as useful as that are coming from a source we can not easily change.